app/javascript/selectors/questionsSelector.js

```diff
@@ -101,11 +101,21 @@ export function getAllActiveQuestionsForCategory(categoryId, buildingId, state)
 
   // Get list of non-dependent questions for the current category
   // that were assigned to this user
-  const rootQuestions = state.categories[categoryId].questions.filter((questionId) => {
-    return buildingQuestionIds.includes(questionId);
-  });
 
-  return getAllActiveQuestions(rootQuestions, buildingId, state);
+  // if in asset manager view categories there is no questions array in the state.categories
```

Comment:
Oops, this is an issue with the way we load asset managers initially - there should be a questions array within state.categories for all types of users (but if I mentioned otherwise please lmk because I remember discussing this with you for some reason :O)

The way to fix this is to go to `application_helper.rb`, and in `asset_manager_initial_state`, change the categories value to this:

```
categories: ActiveModel::Serializer::CollectionSerializer.new(
        Category.all, each_serializer: CategorySerializer
      )
```

this way categories are serialized to include a list of question ids just like with RMI Admins.

Once this is fixed you can go back to the old selector code and assume that `state.categories[categoryId].questions` always exists